src/platform/packages/shared/kbn-saved-search-component/src/components/saved_search.tsx

```diff
@@ -72,6 +73,7 @@ export const SavedSearchComponent: React.FC<SavedSearchComponentProps> = (props)
               searchSourceJSON,
             },
             columns,
+            sort: getDefaultSort(dataView, undefined, undefined, isOfAggregateQueryType(query)),
```

Comment:
This is the `SavedSearchComponent` bug. We simply weren't passing a `sort` to the embeddable so results were unsorted.